### PR TITLE
fix(MongoBinaryDownloadUrl.ts): Add current openSUSE Leap release version

### PR DESCRIPTION
**Pull request by jloehel**
Add current openSUSE Leap release version (15) to the regular
expression. The change should fix the creation of the DownloadURL for
openSUSE Leap 15.x.

Only tested on openSUSE Leap 15.3.

Signed-off-by: Jürgen Löhel <juergen@loehel.de>

### CODE REVIEW

**Comment by nodkz** [comment]
Thanks 🙏

**Comment by hasezoey** [comment]
@nodkz from what i can tell, this was merged into the wrong branch (it was merged to beta, which is currently behind master)

**Comment by hasezoey** [comment]
this PR has been merged originally to branch `beta` (which was behind master), and the commit from beta has been cherry-picked over to master as commit a9ccdbfb1fc20d7322d15a01d3e02919aab5a3c9

---

@jloehel next time please verify your signature on the commit you have if you have signatures enabled (the commit 878e94464990487f5864be4c76dc0cd073dc0c9f is "unverified" because the email from the signature and the commit mismatch)

**Comment by jloehel** [comment]
@hasezoey Ooops, sorry. The next time I'll take care to sign with the correct mail. What is the correct branch? The CONTRIBUTING.md says it's the `next` branch but I could not find that branch. I thought master is only the release branch. Should I open future PRs against master?

**Comment by hasezoey** [comment]
> I thought master is only the release branch. Should I open future PRs against master?

you had correctly assumed to do it against beta, but beta is currently behind master (because beta is currently really only used when staging multiple changes / next major version, instead of always), because it is currently easier to handle that way

TL;DR: you did it correct, but before merge, the one how was going to merge should have checked that the branch was actually correct and up-to-date

thanks for pointing this out, i will try to change it then (currently it is mostly a copy-paste from angular for the commit message style)
